.github/workflows/package-test.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9, 3.10]
```

Comment:
GitHub actions does support py 3.10. You merely need to declare py versions as strings instead of numbers. E.g.:

```
python-version: ["3.7", "3.8", "3.9", "3.10"]
```
The problem you're encountering, is (presumably) that YAML parsing is trimming trailing zeros from numbers. Here's a similar issue I stumbled upon recently:
https://github.com/actions/setup-python/issues/160#issuecomment-724485470